src/components/LcRadio/__tests__/LcRadioGroup.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { mount } from '@vue/test-utils'
+import { LcRadioGroup, LcRadio } from '../index'
+
+let wrapper: any
+
+beforeEach(() => {
+  wrapper = mount(LcRadioGroup, {
+    props: {
+      options: [
+        { label: 'Monsieur', value: 'mr' },
+        { label: 'Madame', value: 'ms' },
+        { label: 'Non spécifié', value: 'unspecified' },
+      ],
+      name: 'civility',
+      modelValue: '',
+    },
+    global: {
+      components: { LcRadio },
+    },
+  })
+})
+
+afterEach(() => {
+  wrapper?.unmount()
+})
+
+describe('LcRadioGroup', () => {
+  it('is a Vue instance', () => {
+    expect(wrapper.vm).toBeTruthy()
+  })
+
+  describe('input behaviour', () => {
+    it('should render good number of radio-buttons', () => {
+      const radiosButtons = wrapper.findAll('input[type="radio"]')
+
+      expect(radiosButtons).toHaveLength(3)
+    })
+
+    it('should set good name attribute', () => {
```

Comment:
les deux tests sont identiques ?
should set good name attribute
should set good value attributes